docker-compose.yml

```diff
@@ -1,42 +1,47 @@
-nginx:
-    image: nginx
-    ports:
-        - "8080:80"
-    volumes:
-        - ./docker/nginx/nginx.conf:/nginx.conf
-        - ./docker/logs/nginx:/var/log/nginx
-        - .:/var/www/html
-    links:
-        - php:php
-    command: nginx -c /nginx.conf
-php:
-    build: docker/php
-    ports:
-        - "9000:9000"
-    volumes:
-        - .:/var/www/html
-    #links:
-        # - "postgres:rdbms"
-        # - "mariadb:rdbms"
-    env_file:
-        - ./docker/php/env
-        # Comment non-used DBMS lines
-        # If all DBMS are commented out, sqlite will be used as default
-        # - ./docker/postgres/env
-        # - ./docker/mariadb/env
-#postgres:
-#    image: postgres:9
-#    ports:
-#        - "5432:5432"
-#    volumes:
-#        - ./docker/data/pgsql:/var/lib/postgresql/data
-#    env_file:
-#        - ./docker/postgres/env
-#mariadb:
-#    image: mariadb:10
-#    ports:
-#        - "3306:3306"
-#    volumes:
-#        - ./docker/data/mariadb:/var/lib/mysql
-#    env_file:
-#        - ./docker/mariadb/env
+version: '2'
+services:
+    nginx:
+        image: nginx
+        ports:
+            - "8080:80"
+        volumes:
+            - ./docker/nginx/nginx.conf:/nginx.conf
+            - ./docker/logs/nginx:/var/log/nginx
+            - .:/var/www/html
+        links:
+            - php:php
+        command: nginx -c /nginx.conf
+    php:
+        build:
+            context: docker/php
+            args:
+                timezone: 'Europe/Monaco'
```

Comment:
It could be a good idea to drop a comment line on the composer file to explain that